batch-launch/batch-launch.py

```diff
@@ -120,8 +121,8 @@ def false_or_empty(l):
 
 # Launch container with each of the things
 print("\nBulk launching {} containers for each of the {}.".format(wrapperName, thingString))
-launchArgs = [{rootElementName: launchArgTemplate.format(thing)} for thing in results]
-url = host + '/xapi/projects/{}/wrappers/{}/bulklaunch'.format(project, wrapperId)
+launchArgs = {rootElementName: json.dumps([launchArgTemplate.format(thing) for thing in results])}
+url = host + '/xapi/projects/{}/wrappers/{}/root/{}/bulklaunch'.format(project, wrapperId, rootElementName)
```

Comment:
These two changed lines are the core of the fix. Everything else is bumping version numbers and rewriting the command json in the image labels.